generator.yaml

```diff
@@ -4,13 +4,18 @@ ignore:
     # - IPSet
     - RegexPatternSet
     # - RuleGroup
-    - WebACL
+    # - WebACL
   field_paths:
     - RuleGroup.Rules.Statement.AndStatement
     - RuleGroup.Rules.Statement.OrStatement
     - RuleGroup.Rules.Statement.NotStatement
     - RuleGroup.Rules.Statement.ManagedRuleGroupStatement.ScopeDownStatement
     - RuleGroup.Rules.Statement.RateBasedStatement.ScopeDownStatement
+    - WebACL.Rules.Statement.AndStatement
+    - WebACL.Rules.Statement.OrStatement
+    - WebACL.Rules.Statement.NotStatement
+    - WebACL.Rules.Statement.ManagedRuleGroupStatement.ScopeDownStatement
+    - WebACL.Rules.Statement.RateBasedStatement.ScopeDownStatement
 model_name: wafv2
 empty_shapes:
   - All
```

Comment:
I'm glad we landed this feature. The more i think about it the more it sounds right.